src/app/business/dataflow/migration.component.ts

```diff
@@ -60,7 +61,7 @@ export class MigrationListComponent implements OnInit {
     ) {
         this.showAKSKWarning = window['akskWarning'];
         this.errorMessage = {
-            "name": { required: "Name is required.",isExisted:"Name is existing" },
+            "name": { required: "Name is required.",isExisted:"This name already exists." },
```

Comment:
Can we call it: "Migration policy with same name already exists"

Reply:
This error appears on the dialog when we are creating a migration policy in the migration section. 
IMO this should be alright as contextually the user knows where they are and what name they are entering.

Reply:
Ok. But in my opinion, these messages are not only for realtime display but these messages can be used for analytics too